README.md

```diff
@@ -76,3 +76,68 @@ Make sure you read the `README.md` file in the `reporting` directory and install
 ## Implementing the benchmark for a new tool
 
 To implement a tool, you need to create a new directory in the solutions directory and give it a suitable name.
+
+## Running the benchmark with Docker
+
+Instructions for running the benchmark starting with a fresh Ubuntu 20.04 VM.
+
+### Configuration
+
+In case of Amazon EC2, create a file system and mount it. Example, assuming the `/dev/nvme1n1` device mounting to `/mnt/data`:
+
+```bash
+sudo mkfs.ext4 /dev/nvme1n1
+sudo mount /dev/nvme1n1 /mnt/data
+```
+
+Install Docker with `apt`:
```

Comment:
Which repo are we using? If we're using the official ones, we might as well put a link here to the instructions:

```md
Install [Docker](https://docs.docker.com/engine/install/ubuntu/#install-using-the-repository) with `apt`:
```

Reply:
We could but the bit about `usermod` is quite sloppy in the official docs:
```
If you would like to use Docker as a non-root user, you should now consider adding your user to the “docker” group with something like:

  sudo usermod -aG docker your-user

Remember to log out and back in for this to take effect!
```

I'd rather do the `gpasswd` + `newgrp` as documented in this README (this took a fair bit of digging on SO and experimenting to get right).

Reply:
I see. Can we mention just the repo part, and then ask the reader to follow the instructions in our README instead?

Reply:
For Ubuntu 20.04, the Docker version in the Ubuntu repository is quite fresh so it works well. I've added a clarification for this and a pointer to the official Docker repo.